cwf/cloud/go/services/cwf/test_init/test_service_init.go

```diff
@@ -18,16 +18,18 @@ import (
 
 	"magma/cwf/cloud/go/services/cwf"
```

Comment:
For all the test service files: as discussed offline, we don't need/want to attach SwaggerSpec servicers to these test services. We'll just spin up ad hoc / anonymous services when we're testing the spec polling functionality

For context, the reason we do attach certain service mesh servicers to these types of test servicers is for when the servicers perform some functionality that is expected/required under normal testing scenarios

Reply:
Got it, this makes a lot more sense